core/src/main/java/org/stellar/anchor/sep31/Sep31Service.java

```diff
@@ -191,10 +194,20 @@ void updateAmounts() throws AnchorException {
   void updateTxAmountsBasedOnQuote() throws AnchorException {
     Sep38Quote quote = Context.get().getQuote();
     if (quote == null) {
+      infoF("Quote for transaction ({}) not found", Context.get().getTransaction().getId());
       throw new ServerErrorException("Quote not found.");
     }
 
     Sep31Transaction txn = Context.get().getTransaction();
+    infoF("Updating transaction ({}) with quote ({}) - inAsset ({}) inAmount ({}) " +
```

Comment:
This has details. Suggest to remove the detail in `infoF` and use `debugF` with details.